src/main/resources/assets/localization/en-us.json

```diff
@@ -396,7 +401,7 @@
   "Gold": "Gold",
   "Lapis": "Lapis",
   "Rose": "Rose",
-  "GunPowder": "GunPowder",
+  "Gunpowder": "Gunpowder",
   "Slime": "Slime",
   "glass": "glass",
   "cloth": "cloth",
```

Comment:
Make sure to modify all changed localisation entries here.

Reply:
Yes, I have used find & replace in the whole project.

Reply:
No, you haven't changed all of them.

Reply:
I have.
![image](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/74168521/77283175-3858-47f2-9492-88f0353e4e36)

Reply:
If you look at where the comment is put, you would notice that gunpowder is the entry I am not talking about.
![bilde](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/37084190/55df0f26-e5d3-46bb-8626-8fd3bebde2f2)

Reply:
I made the changes because the localization keys are using capitalized names.

Reply:
You haven't made any changes to those entries.

Reply:
The original localization entries are captitalized.

Reply:
They don't exist!

Reply:
They do.
![image](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/74168521/acc5e326-0a53-478e-80e0-7b163461afe6)

Reply:
Okay, good